page_v2.go

```diff
@@ -210,9 +215,16 @@ func (dp *dataPageWriterV2) write(ctx context.Context, w io.Writer) (int, int, e
 	if err != nil {
 		return 0, 0, errors.Wrapf(err, "compressing data failed with %s method", dp.codec)
 	}
+
+	var crc32Checksum *int32
+	if dp.enableCRC {
+		v := int32(crc32.ChecksumIEEE(append(append(rep.Bytes(), def.Bytes()...), comp...))) // TODO: is this correct?
```

Comment:
if we're not sure this is correct can we make sure somehow?

Reply:
I validated this as far as I could (against our own implementation and using `parquet-cli`), so I'm fairly the confident this is correct, so I removed the comment.